### ISSUE TITLE
Friendly Actor cannot make Saving Through if the origination item it's Equipment

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#1092]
In GitLab by @rmionescu on Dec 8, 2022, 21:17

I have created a shield that have an attack as an action which requires the target to make a save. 
If I use that action towards a friendly actor the saving throw fails and there are some nasty errors in the game console.

Above error happens only if the item it's equipment type and only if it is used on friendly actors.

Attached the equipment and the error from console:

![midi-console-error](https://my-gitlab-bucket.s3.amazonaws.com/640711674/e40af9ccf9c3d3d8077c855d45a3b13205631bd3b13e8a161b0256b000ad4111/midi-console-error.png)

**Comment by tposney** [comment]
In GitLab by @thatlonelybugbear on Dec 9, 2022, 24:08

Yeah that is an easy fix using optional chaining in `workflow.js:2181` as the midiproperties are not available for equipment
```js
if (isFriendly &&
	(this.saveItem.system.description.value.toLowerCase().includes(i18n("midi-qol.autoFailFriendly").toLowerCase())
              || this.saveItem.flags.midiProperties?.autoFailFriendly))    //in this line add the ? as shown
```

**Comment by tposney** [comment]
Fix in 10.0.24
